platform/ios/CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash when a style layer `*-pattern` property evaluates to nil for a particular feature. ([#12896](https://github.com/mapbox/mapbox-gl-native/pull/12896))
 * Fixed an issue with view annotations (including the user location annotation) and the GL map lagging relative to each other. ([#12895](https://github.com/mapbox/mapbox-gl-native/pull/12895))
 
+### User location
+* Added `-[MGLMapViewDelegate mapViewUserLocationAnchorPoint:]`  to offset the user location annotation with a custom `CGPoint`. ([#12907](https://github.com/mapbox/mapbox-gl-native/pull/12907))
+* Marked `-[MGLMapView setUserLocationVerticalAlignment:]` as deprecated. Use `-[MGLMapViewDelegate mapViewUserLocationAnchorPoint:]` instead. ([#12907](https://github.com/mapbox/mapbox-gl-native/pull/12907))
+* Added `-[MGLMapView updateUserLocationAnnotationView]` and `-[MGLMapView updateUserLocationAnnotationView:animated:]` to public API to update the position of the user location annotation. ([#12907](https://github.com/mapbox/mapbox-gl-native/pull/12907))
```

Comment:
As far as the user is concerned, these methods never existed in the first place, so “to public API” is redundant.

> Added the `-[MGLMapView updateUserLocationAnnotationView]` and `-[MGLMapView updateUserLocationAnnotationView:animated:]` methods to update the position of the user location annotation between location updates.